### ISSUE TITLE
Articulated tractor only gets Bunker Silo mode

### ISSUE THREAD

**Issue by GreenMtns** [Courseplay/Courseplay_FS22#3312]
[Feature]

Could we have a way of choosing witch type of course to set up when using an articulated tractor? Between Bunker Silo and Fieldwork.   

Would like to be able to choose witch type of course is wanted.

Even if the course is set up using a standard tractor it will switch it an articulated tractor is then put on the implement.

Thank You.

**Comment by Tensuko** [comment]
Screenshot of your Tractor with attached tools.

What tractor is it?

Becouse that sounds more like a mod issue then a CP issue.

**Comment by GreenMtns** [comment]
I was able to create a Fieldwork course after uninstalling Farm Sim and reinstalling it but when I tried to bring up the mini hud this is what I get. It would be nice with this articulated tractors to have a choice between the field and silo work.
here is the picture in the field with a harrow set behind an articulated tractor but the hud won't show field work.

![Silo work in the field](https://github.com/Courseplay/Courseplay_FS22/assets/74217253/dc266c82-9501-4e51-8b92-330d0a20ce1c)

**Comment by Tensuko** [comment]
Is this MP Server?
Again: What tractor? Mod? What tool? Mod?

**Comment by GreenMtns** [comment]
My apologies, It has to be a mod somewhere. I stripped the folder to just the tractor which is a mod and the harrows which is also a mod and Course Play works just fine. I am now replacing all mods back into the folder one by one to try to find which one is causing the issue.

Again, my apologies.
________________________________

**Comment by Tensuko** [comment]
No Problem.

**Comment by Szaszabi77** [comment]
I also have the same issue. On a field I could set a course for harvesting in contract job snd worked perfectly. Next step same field also contract job a cultivating. Tractor is a mod named Raba steiger (articulated)
Attachment is also a mod one. At this setup courseplay just wants to set silo work. I tried the method from above so taken all the mods. I left only courseplay newest from github the tractor and the attachment. Everything is up to date.Same result.
Btw I tried it on my own field either..
I desperately need help.

thanks in advance
